lib/px/service/kinesis.rb

```diff
@@ -6,6 +6,9 @@ module Service
     module Kinesis
       AWS_DEFAULT_REGION = "us-east-1" # http://docs.aws.amazon.com/general/latest/gr/rande.html#ak_region
       DEFAULT_SHARD_COUNT = 1
+      DEFAULT_KINESIS_ENDPOINT = "https://kinesis.us-east-1.amazonaws.com"
```

Comment:
Can you avoid repeating the region in the default endpoint that's already specified in `AWS_DEFAULT_REGION`?

Reply:
@mwstobo is this because the local Kinesis endpoint does not use that standard?